packages/suite/src/constants/suite/experimental.ts

```diff
@@ -44,11 +51,27 @@ export const EXPERIMENTAL_FEATURES: Record<ExperimentalFeature, ExperimentalFeat
         },
     },
     'nft-section': {
-        title: 'TR_EXPERIMENTAL_NFT_SECTION',
-        description: 'TR_EXPERIMENTAL_NFT_SECTION_DESCRIPTION',
+        title: { id: 'TR_EXPERIMENTAL_NFT_SECTION' },
+        description: { id: 'TR_EXPERIMENTAL_NFT_SECTION_DESCRIPTION' },
     },
     'global-send-receive': {
-        title: 'TR_EXPERIMENTAL_GLOBAL_SEND_RECEIVE',
-        description: 'TR_EXPERIMENTAL_GLOBAL_SEND_RECEIVE_DESCRIPTION',
+        title: { id: 'TR_EXPERIMENTAL_GLOBAL_SEND_RECEIVE' },
+        description: { id: 'TR_EXPERIMENTAL_GLOBAL_SEND_RECEIVE_DESCRIPTION' },
+    },
+    'experimental-networks': {
```

Comment:
hopefully there will never be more than 1 network available or all will be available under one checkbox

Reply:
However, fine by me